forwarder/gateway_cmd.go

```diff
@@ -70,12 +70,16 @@ var (
 		Run:   gatewayDetails,
 	}
 
-	jsonOutput bool
+	jsonOutput     bool
+	pushToThingsIX bool
 )
 
 func init() {
 	GatewayCmds.PersistentFlags().BoolVar(&jsonOutput, "json", false, "Output in json format")
 
+	onboardGatewayCmd.PersistentFlags().BoolVar(&pushToThingsIX, "push-to-thingsix", false, "Push gateway onboard message to ThingsIX")
```

Comment:
Can we make the command "push" or "push-onboarding" in theory you could also push to other services that implement the same API